astropy/coordinates/baseframe.py

```diff
@@ -1227,7 +1228,7 @@ def is_transformable_to(self, new_frame):
 
         Parameters
         ----------
-        new_frame : class or frame object
+        new_frame : `BaseCoordinateFrame` class or instance
```

Comment:
For now, I guess
```suggestion
        new_frame : frame-instance or `BaseCoordinateFrame` subclass
```
But perhaps after #11472 is fixed, this should be `frame-like`?

Reply:
Agreed!